packages/shared/src/db/types.ts

```diff
@@ -78,3 +78,12 @@ export type Pin = BaseModel<'pins'>;
 export type PinType = schema.PinType;
 export type Settings = BaseModel<'settings'>;
 export type PostWindow = BaseModel<'postWindows'>;
+
+// Type guards
+export function isGroup(obj: Channel | Group): obj is Group {
```

Comment:
Would it make sense to use one of the methods that gets type from id here? This feels a little fragile.